BuffBot/dbUnitTest.py

```diff
@@ -6,23 +6,38 @@
 class DatabaseTest(unittest.TestCase):
     # Prepare the relevant table for testing by deleting all rows in it.
     def emptyTable(self, table):
-        self.conn = sqlite3.connect('test123.db')
+        self.conn = sqlite3.connect("test123.db")
         sql = 'delete from {} '.format(table)
         self.conn.execute(sql)
         self.conn.commit()
         self.conn.close()
 
     def setUp(self):
-        self.emptyTable("game_restriction")
-        # Create a Database object we can call functions on.
+        self.emptyTable('game_restriction')
+        self.emptyTable('members')
+        # Create a Database object to call the relevant functions.
         self.db = Database()
         # Insert dummy data into the database.
         self.db.flag_gaming_channel("1", "Hearthstone", 1)
+        self.db.insert_coins(85431603408420864, 4000, 'Lenny#1112')
+        self.db.insert_coins(235892294857916417, 23, 'Christian#1111')
+        self.db.insert_coins(278286021903515656, 20000, 'BuffBot#0334')  # Mocking the bot user.
```

Comment:
Is it possible to undo these changes after the test? Like self.remove_coins after the tests?